apps/web/components/eventtype/EventAvailabilityTab.tsx

```diff
@@ -205,6 +205,8 @@ const EventTypeSchedule = ({ eventType }: { eventType: EventTypeSetup }) => {
       const value = options.find((option) =>
         scheduleId
           ? option.value === scheduleId
+          : isManagedEventType
+          ? option.value === 0
```

Comment:
We want to show "members' default schedule" text if no specific availability schedule was assigned to the MET